packages/react-from-markup/src/rehydrator.ts

```diff
@@ -75,7 +75,13 @@ export default async (
   const roots = Array.from(
     // TODO: allow setting a container identifier so multiple rehydration instances can exist
     container.querySelectorAll("[data-react-from-markup-container]")
-  );
+  ).reduce((acc: Element[], root: Element) => {
```

Comment:
Could've used a filter here but upted for reduce as to guarantee a reduced number of `r.contains()` calls. Otherwise a `roots.some()` call would've run over the entire array without a reducer